src/styles.css

```diff
@@ -411,18 +432,63 @@ footer .funding {
   margin-bottom: 2px;
 }
 
-.registered span, .metrics span {
+.registered span,
+.metrics span {
   font-size: inherit;
 }
 
 h4.work {
   margin-bottom: 0;
 }
 
-.creator-list, .contributor-list, .funder-list {
+.creator-list,
+.contributor-list,
+.funder-list {
   margin-bottom: 10px;
 }
 
 .panel.aggregations {
   margin-bottom: 20px;
 }
+
+
+/* New styles for mobile-friendly cookie bar. */
+
+.MY_BUTTON_WRAPPER_CLASS {
+  white-space: nowrap;
+}
+
+@media screen and (min-width: 376px) {
+  .MY_CONTENT_CLASS {
+    flex: 1 0 180px !important;
```

Comment:
Are the !important flags required in this file? I tend to prefer not to use !important as it can make further debugging difficult. Is there existing styles coming from elsewhere that are requiring the !important?

Reply:
Just an observation.  I noticed that ORCID put their cookie banner across the top of the page.  It might have been easier to have it there.  

![image](https://user-images.githubusercontent.com/4681304/106033177-6ece1c00-609f-11eb-88aa-2001da3eab07.png)

Reply:
I cleaned up as much as possible, removing any !important flags that were not necessary.  The ones I left were necessary to override inline styles generated by the react-cookie-consent module.